frontend/src/components/Description.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+class Description extends Component {
+
+  render() {
+    return (
+      <div>
+        <h2>Description goes here</h2>
+      </div>
+    );
+  }
+}
+
+export default Description;
```

Comment:
You should add a snapshot test for your new component. Ideally every component would have a snapshot

Reply:
Description test added. Jest tests moved to test folder.